corehq/apps/userreports/README.rst

```diff
@@ -2052,6 +2054,64 @@ Field should refer to report column IDs, not database fields.
      }
    ]
 
+Distinct On
+-----------
+Based on columns mentioned in the distinct on clause,
+The top most row is picked in case of duplicates.
+This is different from aggregation in sense that this is done after fetching
+the rows, whereas aggregation is done before selecting the rows.
+
+This is used in combination with a sort expression to have predictable results.
```

Comment:
```suggestion
This should be used in combination with a sort expression to have predictable results, otherwise an arbitrary
row would be returned first.
```

Reply:
this can't be used without a sort expression and the reason is to have predictability.
`is` to `should be` sounds the same?
there is no otherwise since it cant be used without it, yes?

Reply:
gotcha, I didn't realize that. thanks.